ansible-tower-cli/pipeline-example.yml

```diff
@@ -0,0 +1,20 @@
+apiVersion: tekton.dev/v1alpha1
```

Comment:
We don't have those kinda files yep, but as this is a pipeline, it should be apply cleanly.

Reply:
@vdemeester do you think that I should remove the file and keep it just on the README as a example?

Reply:
maybe for now yeah :angel:

Reply:
Done, just removed the pipeline from the PR